### ISSUE TITLE
dist folder not contained in npm package

### ISSUE THREAD

**Issue by VincentBrunet** [ashiguruma/patternomaly#8]
From the package.json:

```json
{
  "main": "dist/patternomaly.js"
}
```

By installing the package:

```bash
npm install patternomaly --save
```

It means that importing the module from javascript will fail, out of the box:

```javascript
// will try to require node_modules/patternomaly/dist/patternomaly.js
import Pattern from 'patternomaly'
```

**Comment by ashiguruma** [comment]
Hey @VincentBrunet. Thanks for letting me know. I'll take a look at getting this sorted over the weekend.

**Comment by ashiguruma** [comment]
Hey Vincent. I've removed the `dist` directory from the ignore list. I've also published an updated tag ([1.2.1](https://github.com/ashiguruma/patternomaly/releases/tag/v1.2.1)) to NPM.

I'd like to check all the possible include methods to make sure the module actually works as intended. Will leave this issue open for now and close once I've had a chance to check.
